### PR TITLE
Memory hot unplug update

### PR DESCRIPTION
**Pull request by jniederm**
@jelkosz @mz-pdm please review

---
This change is [Reviewable](https://reviewable.io/reviews/ovirt/ovirt-site/1356)

### CODE REVIEW

File: source/develop/release-management/features/virt/hot-unplug-memory.html.md [hunk 1]
```diff
@@ -47,8 +50,55 @@ any memory in order to guarantee that hot plugged memory is movable.
...
+
+If next run configuration exists and its memory value is same as size of memory
+of running VM before hot unplug, next run configuration is updated.
+
+![](/images/wiki/memory-hot-unplug-webadmin.png "Memory hot unplug button in 'VM Devices' tab in Administration Portal")
+
+### REST API
+
+Memory hot unplug can be performed by updating a VM with decremented value
+of `/vm/memory` tag. Sometimes it may be required to decrement value of *Physical
```
Comment (mz-pdm) [inline source/develop/release-management/features/virt/hot-unplug-memory.html.md]: "Sometimes" - when exactly?

File: source/develop/release-management/features/virt/hot-unplug-memory.html.md [hunk 2]
```diff
@@ -47,8 +50,55 @@ any memory in order to guarantee that hot plugged memory is movable.
...
 The previously hot plugged memory appears in the form of memory devices in
 *Vm Devices* tab of the given VM.  You can remove any of the devices (assuming
 they have been hot plugged considering the constraints described above) and
 thus remove that amount of memory from the VM.  Once the memory device is
 successfully hot unplugged the device disappears from the device list.
+
+*Physical Memory Guaranteed* is decremented if necessary.
+
+If next run configuration exists and its memory value is same as size of memory
+of running VM before hot unplug, next run configuration is updated.
```
Comment (mz-pdm) [inline source/develop/release-management/features/virt/hot-unplug-memory.html.md]: Maybe worth to reformulate a bit, e.g.:

... and its memory values is the same as the memory size of the running VM before hot unplug, ... ?

Comment (jniederm) [reply source/develop/release-management/features/virt/hot-unplug-memory.html.md]: done

File: source/develop/release-management/features/virt/hot-unplug-memory.html.md (hunk 1 above)
Comment (jniederm) [reply source/develop/release-management/features/virt/hot-unplug-memory.html.md]: done

**Comment by jniederm** [comment]
@sandrobonazzola please merge

**Comment by jniederm** [comment]
@mykaul please merge

**Comment by jniederm** [comment]
@bproffitt @duck-rh @didib @mureinik please merge

**Pull request merged**